index.js

```diff
@@ -2,26 +2,32 @@ import './style.scss';
 import $ from "jquery";
 import validate from "jquery-validation/dist/jquery.validate.min.js";
 
-var d = new Date();
+const d = new Date();
```

Comment:
If the only place this date gets used is when it's inserted into the ajax response I'd be tempted to move it down there so it's closer to where it gets used. (new Date()).toLocaleString() is probably even fine if its a once-off.

Reply:
Good idea, I shall update

Reply:
Done!